src/class-fields/static-private-names.case

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2017 Valerie Young. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: literal private names
+info: |
+  ClassElement:
+    ...
+    static FieldDefinition ;
+
+  FieldDefinition:
+    ClassElementName Initializer_opt
+
+  ClassElementName:
+    PrivateName
+
+  PrivateName:
+    #IdentifierName
+template: default
+---*/
+
+//- fields
+static #x; static #y
+//- privateinspectionfunctions
+  static x() {
+    this.#x = 42;
```

Comment:
this is a very interesting test case.

Maybe I'm being too precious, but I wonder if the function name should be different than the private name.

Reply:
hey @leobalter -- hmmm well if a javascript interpreter confuses property "#x" with property "x", then that javascript interpreter really has a problem.

Reply:
that's a good point! I was afraid of returning a false positive, but I believe it was just some [FUD](https://en.wikipedia.org/wiki/Fear,_uncertainty_and_doubt).